spatialmath/geometry.go

```diff
@@ -157,6 +157,26 @@ func NewGeometryFromProto(geometry *commonpb.Geometry) (Geometry, error) {
 	return nil, ErrGeometryTypeUnsupported
 }
 
+func NewGeometriesFromProto(proto []*commonpb.Geometry) ([]Geometry, error) {
```

Comment:
Do you need an explicit function that handles multiple Geometries? Or should the client and Server Code handle this the way that arms currently do? Either way, I prefer picking one implementation and consolidate.

I would also expect arms to have more than one Geometry, but bases to have one, when we configure the component.

Reply:
The function right about this one `NewGeometryFromProto` is called in a for loop in arm's `client.go`. Is there a reason now to follow that pattern here?

Reply:
I think it's reasonable to have this additional helper function.

*Every* component is shortly going to be becoming a `resource.Shaped`, so it makes sense to have a thing operating directly on the `[]Geometry` that those will all have.

I'm confused why it matters what the typical quantity of geometries is that some of those components may have.

Reply:
If you'd like to do that in a follow-up pr and test, totally ok.

If there is in fact a reason to have a difference, it's not immediately clear to me at least, and maybe a comment attached to the exported functions would be helpful.

Reply:
Cool I'll update the arm to be consistent